exercises/robot-simulator/src/test/java/RobotTest.java

```diff
@@ -204,6 +204,7 @@ public void testAdvancingWhenFacingWestDecreasesXCoordinateByOne() {
     public void testInstructionsToMoveWestAndNorth() {
         final Robot robot = new Robot(new GridPosition(0, 0), Orientation.NORTH);
 
+        //should this be robot.instructions? Canonical data says property used should be "instructions"
```

Comment:
The property names don't have to exactly match the canonical data as long as the properties do the same thing in my opinion 🙂 What do you think @exercism/java?

And just for future reference, you could always put a comment on the PR instead of in the code, that might be easier to remove 🙂

Reply:
I'd agree with you @FridaTveit - I think as long as they're similar and the property name clearly explains what it will do, I'm happy with slight deviation :slightly_smiling_face: